src/pages/index.tsx

```diff
@@ -21,16 +31,103 @@ import tClientNext from "../tClientNext";
 import PublicUser from '../shared/publicModels/PublicUser';
 import PublicGroup from '../shared/publicModels/PublicGroup';
 import { MdVideocam } from 'react-icons/md';
+import { HSeparator } from 'horizon-ui/components/separator/Separator';
+import { IYuanjianUser } from 'shared/user';
+import { toast } from "react-toastify";
 
 const AppIndex: NextPageWithLayout = () => {
   const user = useUserInfo();
+  if(!user.name){
```

Comment:
spaces needed before ( and after ). alternatively, install formatter to automatically format changed portion of a source file on file saving.

Reply:
ESlint installed

Reply:
I don't think ESlink is primarily used for code formatting -- at least it didn't fix formatting issues with this PR :)